tests/run-tests

```diff
@@ -130,24 +119,38 @@ _up() {
 }
 
 _playbook() {
-    case $mode in
-        host)
-            inventory=inventory.cfg
-            ;;
-        aux)
-            inventory=../example/inventory.cfg
-            ;;
-        *)
-            echo _playbook called in guest mode
-            return 1
-            ;;
-    esac
-    ansible-playbook \
-        -i $inventory \
-        -e @vars-iri.yaml \
-        -e @../example/secrets.yaml \
-        ../example/playbook.yaml \
-        "$@"
+  build=""
+  extraflags=""
+  for arg in "$*"; do
+    if [[ $arg == "build" ]]; then
+      build="--build"
+      extraflags="-e run_update_script=yes"
+    fi
+  done
+
+  case $mode in
+    host)
+      inventory=inventory.cfg
+      ansible-playbook \
+          -i $inventory \
+          -e @vars-iri.yaml \
+          -e @../example/secrets.yaml \
+          ../example/playbook.yaml ${extraflags}
+      ;;
```

Comment:
Let's simplify by removing host mode. It exists because originally that's all there was; then we started thinking about running ansible on the server itself for real installs, so I added guest mode so that I could test it that way. Now that we've settled on that approach and are using it everywhere, I see no reason to carry the extra complexity anymore.

I think that means `tests/inventory.cfg` can be deleted, as it was only used in host mode.

Reply:
Oh, another reason host mode is still around is that under centos 7 the guest had ansible 2.9, and ansible_test didn't work there. That's no longer a concern, so you should be able to get rid of the switch on `$mode` in line 171-178 now, and allow ansible_test in guest mode.

That reminded me to run ansible_test, and it fails. I think it's false positives, will look into how to suppress them.